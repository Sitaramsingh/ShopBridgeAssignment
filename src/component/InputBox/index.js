import React from 'react';
import TextField from '@material-ui/core/TextField';
import PropTypes from 'prop-types';

function inputType(props) {
    return (
        <TextField
            id={props.id}
            error={props.error}
            style={props.style}
            onChange = {props.onChange}
            variant = {props.variant}
            name = {props.name}
            value={props.value}
            label = {props.label}
            type = {props.type}
            helperText = {props.helpertext}
            onKeyPress={props.onKeyPress}
            InputProps = {props.InputProps}
            InputLabelProps = {props.InputLabelProps}
            defaultValue = {props.defaultValue}
            value = {props.value}
            FormHelperTextProps = {props.FormHelperTextProps}
            classes = {props.classes}
            inputProps = {props.inputProps}
            multiline = {props.multiline}
            placeholder = {props.placeholder}
        />
    );
}

inputType.propTypes = {
    id: PropTypes.string,
    style: PropTypes.object,
    type: PropTypes.string,
    helpertext: PropTypes.string,
    variant: PropTypes.string,
    label: PropTypes.string,
    onChange: PropTypes.func,
    error: PropTypes.bool,
    inputstyle: PropTypes.string,
    name: PropTypes.string,
    onKeyPress: PropTypes.func,
    InputProps: PropTypes.object,
    InputLabelProps: PropTypes.object,
    defaultValue: PropTypes.any,
    value: PropTypes.any,
    FormHelperTextProps: PropTypes.object,
    multiline: PropTypes.bool,
    placeholder: PropTypes.string,
};

export default inputType;