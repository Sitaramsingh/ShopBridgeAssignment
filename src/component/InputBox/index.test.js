import React from 'react';
import { shallow } from 'enzyme';
import InputBox from './index';


describe('Component should be render ', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<InputBox/>);
    expect(component).toMatchSnapshot();
  });
});