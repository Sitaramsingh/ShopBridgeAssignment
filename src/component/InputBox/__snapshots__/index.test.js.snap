// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should be render  should render correctly in "debug" mode 1`] = `<WithStyles(ForwardRef(TextField)) />`;
